scss/tech.scss

```diff
@@ -0,0 +1,69 @@
+$tech-height: 450px;
+$tech-width: 300px;
+$tech-logo-size: 100px;
+
+.tech-collection {
+  display: flex;
+  flex-wrap: wrap;
+  margin: 20px 0;
+  padding: 0 10%;
+}
+
+.tech {
+  align-items: center;
+  background-color: $green;
+  border-radius: 10px;
+  display: flex;
+  flex-direction: column;
+  font-size: $text-medium;
+  height: $tech-height;
+  margin: 25px;
+  padding: 25px;
+  position: relative;
+  text-align: center;
+  width: $tech-width;
+
+  &::before {
+    border-color: $green;
+    border-radius: 10px;
+    border-style: solid;
+    content: '';
+    display: flex;
+    font-size: $text-medium;
```

Comment:
I think there's no need for `display: flex` and `font-size: $text-medium` in here